src/marks/contour.js

```diff
@@ -158,7 +158,12 @@ function contourGeometry({thresholds, interval, ...options}) {
         ? thresholds.range(...(([min, max]) => [thresholds.floor(min), max])(finiteExtent(VV)))
         : typeof thresholds === "function"
         ? thresholds(V, ...finiteExtent(VV))
-        : thresholds;
+        : typeof thresholds === "number"
+        ? ticks(...nice(...finiteExtent(VV), thresholds), thresholds)
+        : isIterable(thresholds)
+        ? [...thresholds]
+        : null;
```

Comment:
Use arrayify here (with type = Array since d3-contour requires is strictly)?

```suggestion
        : arrayify(thresholds, Array);
```